fe/fe-core/src/main/java/com/starrocks/sql/ast/JoinRelation.java

```diff
@@ -12,6 +14,12 @@
     private String joinHint = "";
     private final boolean lateral;
 
+    /**
+     * usingColNames is created by parser
+     * and will be converted to onPredicate in Analyzer
+     */
+    private List<String> usingColNames;
```

Comment:
What's the meaning of `usingColNames` ?

Reply:
usingColNames represents the using used in join, eg select * from t0 inner join t1 using(v1) , v1 is usingColNames